### DISCUSSION TITLE
Working on AMD Ryzen 9 7900 with Proxmox 8.2.4

### DISCUSSION THREAD

**Discussion by Wasca** [isc30/ryzen-gpu-passthrough-proxmox#53] [General]
Hello

I'm another very thankful user of your guide. I can confirm that I was able to get iGPU pass through to my Windows 11 Pro VM using a AMD Ryzen 9 7900 CPU and Proxmox 8.2.4

Below are the screen shots of the proof. 

![image](https://github.com/user-attachments/assets/c145fec7-2e83-4d31-a668-1ed5607045e5)

![image](https://github.com/user-attachments/assets/671b7d12-f3f2-4627-9567-72c81c793771)

![image](https://github.com/user-attachments/assets/4af484d5-8640-4ac3-967b-2064eedca7e7)

At the time of writing this, I downloaded the OFFLINE Video Card Driver for this CPU here [https://www.amd.com/en/support/downloads/drivers.html/processors/ryzen/ryzen-7000-series/amd-ryzen-9-7900.html](https://www.amd.com/en/support/downloads/drivers.html/processors/ryzen/ryzen-7000-series/amd-ryzen-9-7900.html)

![image](https://github.com/user-attachments/assets/0666a026-7774-4873-aff8-869889872f8d)

I did need to run the RadeonResetBugFixService.exe. I followed [these instructions](https://oomza.cutegay.software/inga-lovinde/RadeonResetBugFix#install-instructions) I ended up putting the .exe file in the folder C:\Program Files\Radeon Reset Bug\

Attached below is the vbios I used, remove the (.log).

[vbios_7900.bin.log](https://github.com/user-attachments/files/16397399/vbios_7900.bin.log)

**Comment by Kei84** [comment]
Hello, i have the 9/7900 as well, i keep getting the code 43 error, i am using the USB-C connection to a Lenovo dock for triple monitor display, I have tried this in 8.4.x and 9.x, no go with either version of proxmox. i stripped out the vbios as shown, but it produced two files. i only took the 164e file and used it though since i was getting an error using the mv vbios*.bin /path option. this has not worked with either seabios or the OVMF bios options.
